js/App.js

```diff
@@ -0,0 +1,10 @@
+let table = document.getElementsByTagName('table')[0];
+
+table.addEventListener('click',changeColor);
+function changeColor(event){
+    debugger;
```

Comment:
do not push debugger :)

Reply:
sorry I thought I deleted all debuggers (: